packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -44,6 +44,11 @@ import { nls } from '../../../src/messages';
 import { DeployQueue } from '../../../src/settings';
 import { SfdxPackageDirectories } from '../../../src/sfdxProject';
 import { getRootWorkspacePath } from '../../../src/util';
+import {
```

Comment:
Apart from changing these imports, do you see any other work once the library has been published?

Reply:
It's ready to go once we update those imports!